wavefront/resource_alert.go

```diff
@@ -344,3 +366,41 @@ func validateThresholdLevels(m map[string]string) error {
 	}
 	return nil
 }
+
+func decodeRunbookLinks(rawRunbookLinks []interface{}) (links []string) {
+	for _, link := range rawRunbookLinks {
+		links = append(links, link.(string))
+	}
+	return links
+}
+
+func decodeAlertTriageDashboards(rawDashboards []interface{}) (alertTriageDashboards []wavefront.AlertTriageDashboard) {
+	for _, rawDashboard := range rawDashboards {
+		dashboardData := rawDashboard.(map[string]interface{})
+
+		alertTriageDashboard := wavefront.AlertTriageDashboard{
+			DashboardId: dashboardData[dashboardIDKey].(string),
+			Description: dashboardData[descriptionKey].(string),
+			Parameters:  decodeAlertTriageDashboardParameters(dashboardData[parametersKey].([]interface{})),
+		}
+
+		alertTriageDashboards = append(alertTriageDashboards, alertTriageDashboard)
+	}
+
+	return alertTriageDashboards
+}
+
+func decodeAlertTriageDashboardParameters(parameterData []interface{}) map[string]map[string]string {
+	parameters := make(map[string]map[string]string)
+	if len(parameterData) > 0 {
+		for _, parameterBlocks := range parameterData {
+			for parameterBlockType, parameterBlockValue := range parameterBlocks.(map[string]interface{}) {
+				parameters[parameterBlockType] = make(map[string]string)
+				for parameterKey, parameterValue := range parameterBlockValue.(map[string]interface{}) {
+					parameters[parameterBlockType][parameterKey] = parameterValue.(string)
+				}
+			}
+		}
+	}
+	return parameters
+}
```

Comment:
😆🪺🪺🪺🪆🪆🪆

Reply:
This is the pain of dealing with a `map[string]map[string]string`!